ops/generate-subgraph.js

```diff
@@ -71,6 +71,12 @@ function combineFragments(fragments, isTemplate, addresses, missingAddresses) {
       abis = (yamlLoad.abis || [contract]).map(contractName => {
         const versionNum = versionToNum(version);
 
+        if ((versionNum >= 34) && (contractName === "UGenericScheme")) {
```

Comment:
UGenericScheme is deprecated on 34 . no ?

Reply:
Yes, that's why if a DAO still uses it I have to bring it's ABI from v33

Reply:
we should return error in this case